config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryconnectionconnections.bigqueryconnection.cnrm.cloud.google.com.yaml

```diff
@@ -293,10 +289,6 @@ spec:
                     description: The `projectID` field of a project, when not managed
                       by Config Connector.
                     type: string
-                  kind:
```

Comment:
Sadly we can't do this, this is a breaking API change.  We can't break resources that specify `kind`, even though we (I) introduced it by mistake.